### PR TITLE
Add support to boleto, change to use restify-clients, move from npm to yarn, apply JavaScript Google Style...

### PR DESCRIPTION
**Pull request by felipeplets**
Hello @rafaelverger and @andersonba, 

I see there is not much happening on this library for a while so I decided to update it to latest technologies and add boleto capability. 

Let me know if you are interested in this contribution and maybe in future project maintenance. 

All the best, 
Felipe

### CODE REVIEW

**Comment by rafaelverger** [comment]
Wow! Your code improvements are more than welcome, @felipeplets ... work on this repo wasn't really happening since we've moved from maxipago to pagar.me as our payment gateway, but I will take a look in your changes and comment (if necessary) than I will merge it...
Thank u very much for ur contribution :)

File: lib/models.js [hunk 1]
```diff
@@ -48,6 +48,11 @@ exports.sale = {
   shipping: _saleBillingShippingModel,
   transactionDetail: {
     payType: {
+      boleto: {
```
Comment (rafaelverger) [inline lib/models.js]: I would rather use bankSlip or bankPaymentSlip or bankPayment to this type's name (keep it in english)

**Comment by rafaelverger** [comment]
@felipeplets , check my comments :)

File: lib/models.js (hunk 1 above)
Comment (felipeplets) [reply lib/models.js]: @rafaelverger this is equivalent to the node boleto in the maxipago documentation, it's not an option to call it something else. You can search for "<boleto>" at the documentation to check it http://developers.maxipago.com/apidocs/maxipago/boleto-bancario/.

**Pull request merged**